.solhint.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "solhint:recommended",
```

Comment:
don't think this should be checked in?

Reply:
I saw v4-core also checked in solhint config? happy to remove tho

Reply:
+ remove, we don't use solhint anymore anyways - just forge fmt